docs/adr/ADR-018-codedb-index-only-bleve.md

```
@@ -0,0 +1,151 @@
+```go
+// cmd/ox/code.go:204
+snippet := stripANSIEscapes(r.Content)   // generate ANSI … then strip it
+snippet = compactSnippet(snippet, 120)   // plain ≤120-char snippet is all agents get
+```
+
+So we pay (a) the compute to highlight on every search and (b) ~74% of the Bleve
+index to store content — to produce an ANSI string we immediately discard.
+
+**Measured savings** (`TestBleveStoredContentCost`, 1707 ox `.go` files):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)